modules/tensor_mechanics/src/kernels/StressDivergenceRSphericalTensors.C

```diff
@@ -70,6 +64,8 @@ StressDivergenceRSphericalTensors::calculateJacobian(unsigned int ivar, unsigned
 {
   RealGradient test_r, phi_r;
 
+  mooseAssert(ivar == 0 && jvar == 0, "Invalid component in Jacobian Calculation"); // Only nonzero case for a 1D simulation
```

Comment:
Probably shouldn't be called `ivar` and `jvar` but `icomponent` and `jcomponent` or something like that.